src/lib/services-manager/clients/services_manager.rb

```diff
@@ -243,30 +251,41 @@ def buttons
         HBox(*buttons)
       end
 
+      def start_options_for(service)
+        start_modes = ServicesManagerService.start_modes(service)
+
+        [:on_boot, :on_demand, :manual].each_with_object([]) do |mode, all|
```

Comment:
can these three be constant in ServicesManagerService or SystemService?

Reply:
Actually, sometimes I miss some kind of enum (ala [active hash](https://github.com/zilkey/active_hash)).